packages/common/src/visualizations/CartesianChartDataModel.ts

```diff
@@ -162,9 +162,18 @@ export class CartesianChartDataModel
                 const seriesFormat = Object.values(display?.series || {}).find(
                     (s) => s.yAxisIndex === index,
                 )?.format;
-                const seriesLabel = Object.values(display?.series || {}).find(
-                    (s) => s.yAxisIndex === index,
-                )?.label;
+
+                const singleYAxisLabel =
+                    // NOTE: When there's only one y-axis left, set the label on the series as well
+                    this.fieldConfig?.y.length === 1 &&
```

Comment:
is there a way to get this working without the field config or not? future refactor will mean it's not available here